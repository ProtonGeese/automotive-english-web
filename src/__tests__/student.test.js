import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import HondaStudentNew from '../app/student.jsx';

test('Form contains all elements', () => {
  const component = shallow(
    <MuiThemeProvider>
      <HondaStudentNew/>
    </MuiThemeProvider>
  );

  expect(toJson(component)).toMatchSnapshot();
});
