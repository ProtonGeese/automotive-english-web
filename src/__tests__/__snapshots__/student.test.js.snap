exports[`test Form contains all elements 1`] = `<HondaStudentNew />`;
